beetsplug/lyrics.py

```diff
@@ -671,3 +703,24 @@ def get_lyrics(self, artist, title):
                 self._log.debug(u'got lyrics from backend: {0}',
                                 backend.__class__.__name__)
                 return _scrape_strip_cruft(lyrics, True)
+
+    def append_translation(self, text, to_language):
+        import xml.etree.ElementTree as ET
+
+        if not self.bing_auth_token:
+            self.bing_auth_token = self.get_bing_access_token()
+        if self.bing_auth_token:
+            headers = {"Authorization ": self.bing_auth_token}
+            url = ('http://api.microsofttranslator.com/v2/Http.svc/'
+                   'Translate?text=%s&to=%s' % (text.replace('\n', '|'),
+                                                to_language))
+            r = requests.get(url, headers=headers)
+            if r.status_code != 200:
+                self._log.debug(r.text)
```

Comment:
This should probably be a little more explicit, like:

    self._log.debug('translation API error {}: {}', r.status_code, r.text)